### DISCUSSION TITLE
HttpTransfer PostData iOS only?

### DISCUSSION THREAD

**Discussion by GuidoNeele** [shinyorg/shiny#761] [Q&A] [answered]
Was looking at uploading files using Shiny.Net.Http and want to send some data along with the file, so I have a reference to the uploaded file. But it looks like the PostData property on the HttpTransferRequest is only used on iOS. Is this correct? 

https://github.com/shinyorg/shiny/blob/935eaecb0f4e8a6583d9e151cabeb9a05052d826/src/Shiny.Net.Http/HttpTransferRequest.cs#L30

Android is using the TransferJob and I would expect the PostData to be added to the content portion of the HttpRequestMessage.
https://github.com/shinyorg/shiny/blob/935eaecb0f4e8a6583d9e151cabeb9a05052d826/src/Shiny.Net.Http/Infrastructure/TransferJob.cs#L100-L130

**Comment by aritchie** [comment, marked as answer]
Hmmm seems it was forgotten about probably because I generally just rely on the headers and path for uploads.  In any case, it is easy enough to add

**Reply by aritchie** [reply]
https://github.com/shinyorg/shiny/commit/b745abd48e20ab851fded32d47ca27d5082e1a54
